source/jormungandr/jormungandr/street_network/valhalla.py

```diff
@@ -237,6 +237,11 @@ def check_response(self, response):
     def direct_path(self, mode, pt_object_origin, pt_object_destination, datetime, clockwise, request):
         data = self._make_data(mode, pt_object_origin, [pt_object_destination], request, api='route')
         r = self._call_valhalla('{}/{}'.format(self.service_url, 'route'), requests.post, data)
+        if r is not None and r.status_code != 200:
```

Comment:
even for error the response can be fat ?

my point @TeXitoi if it happens that the test is a bit broad I think it would be nice to at least see it in the logs